tests/kuttl/tests/dataplane-deploy-no-nodes-test/00-dataplane-create.yaml

```diff
@@ -8,28 +8,27 @@ data:
   nova-blank.conf: Zm9vCg==
 ---
 apiVersion: dataplane.openstack.org/v1beta1
-kind: OpenStackDataPlane
+kind: OpenStackDataPlaneNodeSet
 metadata:
-  name: openstack-edpm-no-nodes
+  name: edpm-compute-no-nodes
 spec:
+  preProvisioned: true
+  services:
+    - configure-network
+    - validate-network
+    - install-os
+    - configure-os
+    - run-os
+    - ovn
+    - libvirt
+    - nova
   deployStrategy:
-      deploy: false
```

Comment:
this isn't right. It needs to be created in the 00 file with deploy:false. Then we verify it was created and not deployed, then we patch deploy:true in the 01 file, and verify it got deployed.

Reply:
we can leave it as-is though, since I will be reworking this test in the OpenStackDataPlaneDeployment PR.